pages/workspaces/components/FormCard/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, useColorModeValue } from '@chakra-ui/react';
+
+interface Props {
+  name: string;
+}
+
+export const FormCard: React.FC<Props> = ({ name }) => {
+  const backgroundColor = useColorModeValue('white', 'gray.700');
+  const textColor = useColorModeValue('black', 'white');
+
+  return (
+    <Box
```

Comment:
Cards are kind of bland right now, they will probably end up with a little edit footer section, just wanted to get creation working and wired up.

Reply:
nice - yeah we could shove some basic analytics in there too like response count